cmd/context/create.go

```diff
@@ -133,13 +137,15 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 			transformedCtx := okteto.K8sContextToOktetoUrl(ctx, ctxOptions.Context, ctxOptions.Namespace, c.K8sClientProvider)
 			if transformedCtx != ctxOptions.Context {
 				ctxOptions.Context = transformedCtx
+				ctxOptions.Namespace = ctxStore.Contexts[ctxOptions.Context].Namespace
 				ctxOptions.IsOkteto = true
 			}
 		}
 	}
 
 	if okCtx, ok := ctxStore.Contexts[ctxOptions.Context]; !ok {
 		ctxStore.Contexts[ctxOptions.Context] = &okteto.OktetoContext{Name: ctxOptions.Context}
+		// What should be the namespace here?
```

Comment:
I don't think any namespace should be placed here since this code is to create a new namespace, and we want to set the context if the context already exists